src/IRVisitor.h

```diff
@@ -80,6 +80,7 @@ class IRGraphVisitor : public IRVisitor {
     EXPORT virtual void include(const Stmt &);
     // @}
 
+private:
```

Comment:
Not sure about this change if the `include` methods are still protected. E.g. one might want to write a derived class that prints debugging info on the graph and thus needs access to the set.